src/main/java/org/generationcp/middleware/dao/GermplasmSearchDAO.java

```diff
@@ -1541,20 +1580,51 @@ private boolean addPreFilteredGids(final GermplasmSearchRequest germplasmSearchR
 			.list();
 	}
 
-	public List<UserDefinedField> getGermplasmAttributeTypes(final GermplasmSearchRequest germplasmSearchRequest) {
+	public List<Variable> getGermplasmAttributeVariables(final GermplasmSearchRequest germplasmSearchRequest, final String programUUID) {
```

Comment:
@cuenyad I get why this function is in GermplasmSearchDao but I think that the query itself should be in GermplasmDao receiving a list of GID
This function is not exclusive to the germplasm search anymore.